rmgpy/data/solvation.py

```diff
@@ -132,6 +137,87 @@ def process_old_library_entry(data):
     raise NotImplementedError()
 
 
+def add_solute_data(solute_data1, solute_data2, group_additivity=False, verbose=False):
```

Comment:
I named it "add_solute_data" because there's a function called 'add_thermo_data' in 'rmgpy/data/thermo.py` that essentially does the same thing for thermo data.
I can change to "append_solute_data" if that's more clear.

Reply:
We decided to keep the naming consistent with 'rmgpy/data/thermo.py', so I won't change the function name.